diff --git a/include/freetype/config/ftoption.h b/include/freetype/config/ftoption.h
index 18c7d13e..6bded39c 100644
--- a/include/freetype/config/ftoption.h
+++ b/include/freetype/config/ftoption.h
@@ -649,9 +649,9 @@ FT_BEGIN_HEADER
    *
    * [1] https://www.microsoft.com/typography/cleartype/truetypecleartype.aspx
    */
-/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  1         */
-#define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2
-/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  ( 1 | 2 ) */
+/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  1 */
+/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2 */
+#define TT_CONFIG_OPTION_SUBPIXEL_HINTING  ( 1 | 2 )
 
 
   /**************************************************************************
@@ -771,7 +771,7 @@ FT_BEGIN_HEADER
    * switch between the two engines using the `hinting-engine' property of
    * the type1 driver module.
    */
-/* #define T1_CONFIG_OPTION_OLD_ENGINE */
+#define T1_CONFIG_OPTION_OLD_ENGINE
 
 
   /*************************************************************************/
@@ -814,7 +814,7 @@ FT_BEGIN_HEADER
    * switch between the two engines using the `hinting-engine' property of
    * the cff driver module.
    */
-/* #define CFF_CONFIG_OPTION_OLD_ENGINE */
+#define CFF_CONFIG_OPTION_OLD_ENGINE
 
 
   /*************************************************************************/
