diff --git a/include/freetype/config/ftoption.h b/include/freetype/config/ftoption.h
index 908e4e970..f913ddfa4 100644
--- a/include/freetype/config/ftoption.h
+++ b/include/freetype/config/ftoption.h
@@ -643,8 +643,8 @@ FT_BEGIN_HEADER
    * https://www.microsoft.com/typography/cleartype/truetypecleartype.aspx
    */
 /* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  1         */
-#define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2
-/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  ( 1 | 2 ) */
+/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2         */
+#define TT_CONFIG_OPTION_SUBPIXEL_HINTING  ( 1 | 2 )
 
 
   /**************************************************************************
@@ -761,7 +761,7 @@ FT_BEGIN_HEADER
    * between the two engines using the 'hinting-engine' property of the type1
    * driver module.
    */
-/* #define T1_CONFIG_OPTION_OLD_ENGINE */
+#define T1_CONFIG_OPTION_OLD_ENGINE
 
 
   /*************************************************************************/
@@ -803,7 +803,7 @@ FT_BEGIN_HEADER
    * between the two engines using the 'hinting-engine' property of the cff
    * driver module.
    */
-/* #define CFF_CONFIG_OPTION_OLD_ENGINE */
+#define CFF_CONFIG_OPTION_OLD_ENGINE
 
 
   /*************************************************************************/
